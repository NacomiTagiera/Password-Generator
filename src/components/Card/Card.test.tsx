import { render, screen } from '@testing-library/react';

import { Card } from '.';

describe('Card', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Card>Test</Card>);

    expect(asFragment()).toMatchSnapshot();
  });

  it('displays its children', () => {
    render(<Card>Test</Card>);

    expect(screen.getByText('Test'));
  });
});
