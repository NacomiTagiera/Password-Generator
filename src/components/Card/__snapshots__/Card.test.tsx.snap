// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<DocumentFragment>
  <main
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 css-hho69s-MuiPaper-root"
  >
    Test
  </main>
</DocumentFragment>
`;
